### PR TITLE
adding-edit-to-frontend

### PR DESCRIPTION
**Pull request by microna**

### CODE REVIEW

File: front/src/components/Header.jsx [hunk 1]
```diff
@@ -68,24 +67,7 @@ export const Header = () => {
...
-                    <button
-                      className="text-white bg-blue-700 hover:bg-blue-800 focus:ring-4 focus:ring-blue-300 font-medium rounded-lg text-sm px-5 py-2.5 me-2 mb-2 dark:bg-blue-600 dark:hover:bg-blue-700 focus:outline-none dark:focus:ring-blue-800"
-                      variant="contained"
-                    >
-                      Create account
-                    </button>
-                  </Link>
-                </>
+                <></>
               </>
```
Comment (konantovich) [inline front/src/components/Header.jsx:71]: We can write `null` instead 
`<>
<></>
</>`

File: front/src/page/home/components/TableList.jsx [hunk 2]
```diff
@@ -49,126 +52,32 @@ export const TableList = () => {
             obj.map((item) => {
               return (
                 <tr
-                  onClick={() => {
-                    // navigate("audit/" + item._id);
-                  }}
+                  onClick={() => {}}
```
Comment (konantovich) [inline front/src/page/home/components/TableList.jsx:55]: Its redundant onClick

File: front/src/utils/dateFormatter.jsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import { format } from "date-fns";
+
+export default function formatDate(date) {
+  return format(new Date(date), "dd/MM/yyyy");
+}
```
Comment (konantovich) [inline front/src/utils/dateFormatter.jsx:5]: Nice!

**Review by konantovich — APPROVED** [review]

File: front/src/components/Header.jsx (hunk 1 above)
Comment (microna) [reply front/src/components/Header.jsx:71]: changed

File: front/src/page/home/components/TableList.jsx (hunk 2 above)
Comment (microna) [reply front/src/page/home/components/TableList.jsx:55]: deleted

File: front/src/utils/dateFormatter.jsx (hunk 3 above)
Comment (microna) [reply front/src/utils/dateFormatter.jsx:5]: thanks!

**Pull request merged**